code/XMLTester.xml

```diff
@@ -3621,7 +3621,7 @@
 <data_writer profile_name="writer_xml_conf_resource_limits_profile">
     <topic>
         <resourceLimitsQos>
-            <max_samples>200</max_samples>
+            <max_samples>2000</max_samples>
```

Comment:
A similar change should be made [here](https://github.com/eProsima/Fast-DDS-docs/blob/e67f92daf31d285a6b46feee830a210fc84ad686/code/DDSCodeTester.cpp#L4291-L4292)